selinux/olak.fc

```diff
@@ -0,0 +1,8 @@
+# vim: set ts=8 sw=8 noet:
+/usr/bin/openssh-ldap-authkeys		--	gen_context(system_u:object_r:olak_exec_t,s0)
+/etc/openssh-ldap-authkeys		-d	gen_context(system_u:object_r:olak_config_t,s0)
+/etc/openssh-ldap-authkeys/authmap	--	gen_context(system_u:object_r:olak_config_t,s0)
+/etc/openssh-ldap-authkeys/olak.yml	--	gen_context(system_u:object_r:olak_config_t,s0)
```

Comment:
Couldn't you simplify this to a single rule with `/etc/openssh-ldap-authkeys/.*`?

Reply:
Done - I initially considered setting this context on only the `authmap` and `olak.yml` files but I think it's harmless to set it on everything in this directory.